### PR TITLE
feat: create donation banner type

### PR DESCRIPTION
**Pull request by christian14b**
Donation Banner type work

image

### CODE REVIEW

File: src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+								:href="`/donate/supportus/?dfmode=${frequency}&
+									dfamt=${numeral(buttonAmount).format('0.00')}`"
+							>
+								Donate ${{ buttonAmount }}
+							</kv-button>
+							<kv-button
+								v-kv-track-event="['promo', 'click-other', 'DonationBanner', 0, 0]"
+								variant="secondary"
+								class="other-amount"
+								:href="`/donate/supportus??dfmode=${frequency}&dfmat=0.00`"
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue:67]: `??`

File: src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+								v-kv-track-event="[
+									'promo',
+									'click-amount-btn',
+									'DonationBanner',
+									buttonAmount,
+									buttonAmount
+								]"
+								variant="secondary"
+								class="tw-border-brand tw-text-brand"
+								:href="`/donate/supportus/?dfmode=${frequency}&
```
Comment (mcstover) [inline src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue:58]: `/?` > `?`

File: src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+				<div
+					class="tw-flex tw-flex-col md:tw-flex-row tw-items-center tw-justify-center tw-relative tw-gap-0 md:tw-gap-3"
+				>
+					<div class="tw-mb-2 lg:tw-m-0">
+						<div class="tw-relative" style="width: 8rem; height: 8rem;">
+							<kv-contentful-img
+								v-if="bannerImageUrl"
+								class="tw-absolute tw-h-full tw-w-full tw-z-1"
+								:contentful-src="bannerImageUrl"
+								alt=""
```
Comment (dyersituations) [inline src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue:16]: Should we add an alt here?

File: src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+							>
+								Other
+							</kv-button>
+						</div>
+						<button
+							v-kv-track-event="[
+								'promo',
+								'click-donation-banner-close',
+								'x',
+							]" class="`
```
Comment (dyersituations) [inline src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue:77]: The attributes should go on newlines

File: src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+								:href="`/donate/supportus??dfmode=${frequency}&dfmat=0.00`"
+							>
+								Other
+							</kv-button>
+						</div>
+						<button
+							v-kv-track-event="[
+								'promo',
+								'click-donation-banner-close',
+								'x',
```
Comment (dyersituations) [inline src/components/WwwFrame/PromotionalBanner/Banners/Donation/DonationBanner.vue:76]: Is this an appropriate value here?

**Review by emuvente — APPROVED** [review]

**Pull request merged**